src/NUnitFramework/framework/Constraints/ConstraintResult.cs

```diff
@@ -1,4 +1,4 @@
-﻿// ***********************************************************************
+// ***********************************************************************
```

Comment:
Please revert and amend so that your PR doesn't show up in the history of this file.